malariagen_data/anopheles.py

```diff
@@ -2979,7 +2978,7 @@ def biallelic_diplotype_pairwise_distances(
         sample_sets: Optional[base_params.sample_sets] = None,
         sample_query: Optional[base_params.sample_query] = None,
         sample_indices: Optional[base_params.sample_indices] = None,
-        site_mask: Optional[base_params.site_mask] = None,
+        site_mask: Optional[base_params.site_mask] = base_params.DEFAULT,
```

Comment:
Will cause behaviour change for `biallelic_diplotype_pairwise_distance()` when `site_mask` parameter is not given.